theme/static/css/hydroshare_core.css

```diff
@@ -3438,6 +3450,15 @@ span.you-flag {
   animation-duration: 0.6s; /* So they look staggered */
 }
 
+.fade-enter-active, .fade-leave-active {
+	transition: opacity .35s;
+	transition-timing-function: ease;
```

Comment:
Is .35s a setting we should const/standardize/parameterize across similar elements ?

Reply:
This is the animation time that bootstrap uses for collapse transition. This animation was put in place to replace the previous bootstrap collapse because it was not working inside the Vue instance and I could not figure out why. So, to keep it a little consistent I used the same time.